src/core/settings/qgssettings.h

```diff
@@ -79,6 +80,37 @@ class CORE_EXPORT QgsSettings : public QObject
       Gps, //!< GPS section, since QGIS 3.22
     };
 
+#ifndef SIP_RUN
+
+    /**
+     * Returns the tree root element for the settings
+     * \since QGIS 3.30
+     */
+    static QgsSettingsTreeElement *treeRoot();
+
+    static inline QgsSettingsTreeElement *sTtreeApp = treeRoot()->createChildElement( QStringLiteral( "app" ) );
+    static inline QgsSettingsTreeElement *sTtreeConnections = treeRoot()->createChildElement( QStringLiteral( "connections" ) );
```

Comment:
```suggestion
    static inline QgsSettingsTreeElement *sTreeApp = treeRoot()->createChildElement( QStringLiteral( "app" ) );
    static inline QgsSettingsTreeElement *sTreeConnections = treeRoot()->createChildElement( QStringLiteral( "connections" ) );
```

or was there a reason for this?